examples/flower_classifier/train.py

```diff
@@ -0,0 +1,70 @@
+import click
+
+from keras_image_classifier import MLflowInceptionV3, KerasImageClassifier, MLflow_VGG16
+
+import os
+
+
+def download_input():
+    import requests
+    url = 'http://download.tensorflow.org/example_images/flower_photos.tgz'
+    r = requests.get(url)
+    with open('flower_photos.tgz', 'wb') as f:
+        f.write(r.content)
+    import tarfile
+
+    with tarfile.open("flower_photos.tgz") as tar:
+        tar.extractall(path="./")
+
+
+@click.command(help="Trains an Keras model on flower_photos dataset."
+                    "The input is expected as a directory tree with pictures for each category in a"
+                    " folder named by the category."
+                    "The model and its metrics are logged with mlflow.")
+@click.option("--epochs", type=click.INT, default=1, help="Maximum number of epochs to evaluate.")
+@click.option("--batch-size", type=click.INT, default=1,
+              help="Batch size passed to the learning algo.")
+@click.option("--seed", type=click.INT, default=97531, help="Seed for the random generator.")
+@click.option("--training-data")
+@click.option("--test-ratio", type=click.FLOAT, default=0.2)
+@click.option("--pretrained-weights", type=click.STRING, default="None")
+@click.option("--model-type", type=click.STRING, default="VGG16")
+def run(training_data, test_ratio, epochs, batch_size, seed, pretrained_weights, model_type):
```

Comment:
Can we insert a log statement here that indicates which parameters are being used for training? i.e. the value of `epochs`, `batch_size`, `model_type`, `pretrained_weights`, etc. This will be a useful sanity check for the user before they accidentally spend minutes/hours training a model with the wrong parameters.

Reply:
The train method of the classifier starts a new run and logs all the params

Reply:
ah I see, yeah that makes sense